aFWall/src/main/java/dev/ukanth/ufirewall/util/FingerprintUtil.java

```diff
@@ -115,26 +115,18 @@ void setFullscreenDialog(){
             try{
 
                 WindowManager manager = (WindowManager) getContext().getSystemService(Activity.WINDOW_SERVICE);
-                int width, height;
-
-                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.FROYO) {
-                    width = manager.getDefaultDisplay().getWidth();
-                    height = manager.getDefaultDisplay().getHeight();
-                } else {
-                    Point point = new Point();
-                    manager.getDefaultDisplay().getSize(point);
-                    width = point.x;
-                    height = point.y;
-                }
+
+                Point point = new Point();
+                manager.getDefaultDisplay().getSize(point);
```

Comment:
Yes I know, I chose that because what you are mentioned are deprecated. (reported to me by Android Studio)
The best thing is to select what it is not deprecated, Right? :)